test/unit/utils/dataProvider.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
     ),
     CaseEvent: getCaseEventDefinitions([
       'CaseEvent',
+      'CaseEvent-alt-service-more-case-forward-prod',
```

Comment:
Can you please confirm if this should be feature toggled at all?

Alt service has been already released. I don't know about any other release for this epic. We have very few release slots by the end of project and I don't know anything about `alt-service-more-case-forward`.

Reply:
My understanding was that everything new had to be behind a feature toggle. As this didnt align up with any feature toggles that i was aware of, i used the epic name from the ticket. 

How would you have done it?